<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlForm</class>
 <widget class="QWidget" name="ControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>43</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btn_info">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>ðŸ›ˆ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_back">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>â—€ </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_pause">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>||</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_forward">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>â–¶</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_fullscreen">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>ðŸ–¥</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_opacity">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>ðŸ‘»</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_menu">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>ðŸ”§</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_exit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_forward</tabstop>
  <tabstop>btn_back</tabstop>
  <tabstop>btn_pause</tabstop>
  <tabstop>btn_fullscreen</tabstop>
  <tabstop>btn_opacity</tabstop>
  <tabstop>btn_menu</tabstop>
  <tabstop>btn_exit</tabstop>
  <tabstop>btn_info</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
